### DISCUSSION TITLE
[PSD1] Exercise 1.23

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#124] [Esercizi (A-L)]
**Exercise 1.23** Convert the following sign/magnitude numbers to decimal
(a) $1010_2$
(b) $110110_2$
(c) $01110000_2$
(d) $10011111_2$

- _**N.B.**: I numeri da convertire sono gli stessi dell'Esercizio 1.21 (#123 ) ma questa volta sono da assumere in rappresentazione sign/magnitude anziché in rappresentazione in complemento a due._

**Comment by niccolo-fato** [comment]
![photo_5870846455839833126_y](https://user-images.githubusercontent.com/73995334/210087618-7c6821bd-cd66-4d42-ba52-4a83ad909717.jpg)
